src/foam/core/Proxy.js

```diff
@@ -224,7 +224,8 @@ foam.CLASS({
               return this[property][name].apply(this, arguments);
             } :
             function forward() {
-              return this[property][name].apply(this[property], arguments);
+              if ( this[property] && this[property][name] )
```

Comment:
Where/when is this happening?

Reply:
It's happening when I click on the 'next' button on schedule deposit review page of the load funds flow. Here below.
<img width="577" alt="image" src="https://user-images.githubusercontent.com/55885868/196277374-400b0a56-0bff-497c-9075-a1e799dc7b57.png">